src/test/java/org/takes/facets/flash/RsFlashTest.java

```diff
@@ -43,12 +46,25 @@
      */
     @Test
     public void addsCookieToResponse() throws IOException {
+        final String msg = "hey, how are you?";
         MatcherAssert.assertThat(
             new RsPrint(
-                new RsFlash("hey, how are you?")
+                new RsFlash(msg)
             ).print(),
-            Matchers.containsString("Set-Cookie: RsFlash=hey, how are you?")
+            Matchers.containsString(
+                new StringBuilder("Set-Cookie: RsFlash=")
+                    .append(
+                        DatatypeConverter.printBase64Binary(
+                            new StringBuilder(
+                                Level.INFO.getName()
+                            )
+                                .append('/')
```

Comment:
@dmzaytsev not sure about the indentation here - maybe move this line up along with closing `)`. @yegor256 , what do you say?